src/ucs/config/parser.c

```diff
@@ -1115,59 +1118,42 @@ int ucs_config_sscanf_key_value(const char *buf, void *dest, const void *arg)
         ucs_string_split(token, VALUE_DELIM, 2, &key, &value);
 
         if (NULL == value) {
-            common_value = key;
+            /* No value means key is the default value for all */
+            default_value = key;
         } else {
-            if ((key_index = ucs_config_key_find(param->keys, key)) < 0) {
-                goto err;
+            /* Specific key config is present */
+            if ((index = ucs_config_key_find(param->keys, key)) < 0) {
+                return 0;
```

Comment:
shall we print an error?

Reply:
Well, initially I had an error here:
```
ucs_error("Key '%s' is not supported", key);
```
But you suggested to remove it (`no need to print another error`): https://github.com/openucx/ucx/pull/9555#discussion_r1431286438

Should I put it back?

Reply:
yes: the previous function ucs_config_key_value_set() used to print an error, but the new function ucs_config_key_find() does not print an error anymore.

Reply:
Actually `ucs_config_key_value_set()` [didn't](https://github.com/openucx/ucx/pull/9555/commits/a925c6df4b72769fb8b4fbceba207e5086c8d198) print an error, but just returned a status.

Anyway, sure, I will add it back